app/index.py

```diff
@@ -1,24 +1,38 @@
 import easygui
 import NewEggService
+from TonerLandService import TonerLandService
 from flask import Flask
 from searchWindows import searchWindows
 
+
+
 app = Flask(__name__)
 
 @app.route('/')
 
+# def index():
+#     testResponse = NewEggService.GetResults()
+#     for excelObj in testResponse:
+#         print(excelObj.name)
+#         print(excelObj.price)
+#         print(excelObj.url)
+#         print(excelObj.vendor)
+#     return 'asdf'
+  
 def index():
-    testResponse = NewEggService.GetResults()
-    for excelObj in testResponse:
-        print(excelObj.name)
-        print(excelObj.price)
-        print(excelObj.url)
-        print(excelObj.vendor)
-    return 'asdf'
+    tls = TonerLandService()
+    #test = tls.createUrlArray()
+    #results=tls.GetResults()
+    # print(test)
+    results = tls.findContainers()
```

Comment:
findContainers is always returning 0.

Reply:
changed the return value to the list of urls.